test/gen_integration.sh

```diff
@@ -287,7 +287,7 @@ languageVariables:
           echo 'Curling service IP'
           curl -m 3 \$SERVICEIP:$serviceport
           kill \$tunnelPID
-      - run: ./draft -b main -v generate-workflow -d ./langtest/ -c someAksCluster -r someRegistry -g someResourceGroup --container-name someContainer
+      - run: ./draft -b main -v generate-workflow -d ./langtest/ -c someAksCluster -r someRegistry -g someResourceGroup --container-name someContainer --deploy-type helm
```

Comment:
this looks good! can we add a check that the yamls in the .github folder are valid yaml just to be sure the generation can't produce blank fields/newlines? otherwise looks ready to merge

Reply:
I've added validation for the generated yaml files. Let me know if it looks good.